apps/tangle-dapp/constants/evm.ts

```diff
@@ -144,3 +144,45 @@ export const updatePaymentDestination = async (
 
   return txHash;
 };
+
+export const unBondTokens = async (
+  nominatorAddress: string,
+  numberOfTokens: number
+): Promise<`0x${string}`> => {
```

Comment:
I feel like type `0x${string}` is a type that is used a lot through out our dapps and libs, maybe we can create this type somewhere and reuse it.

Reply:
I'm addressing this in my current working PR.